doc/guides/test/sanitycheck.rst

```diff
@@ -390,6 +395,17 @@ harness_config: <harness configuration options>
              tags: sensors
              depends_on: i2c
 
+    The following is an example yaml file with pytest harness_config options, default pytest_root
+    name will be used if pytest_root not specified.
+
+    ::
+
+        tests:
+          test_section:
```

Comment:
This field is used as a test name, and the name `test_section` might be confusing. Also, the indentation is missing for the block below.

Reply:
FIY this is the error in the output for this (after adding indentation below):
```bad test name 'test_section' in /home/maciej/zephyrproject2/zephyr/tests/pytest/.. Tests should reference the category and subsystem with a dot as a separator```

Reply:
fixed in #33227